samples/cpp_with_xamarin/README.md

```diff
@@ -595,7 +595,7 @@ Copying libMathFuncs.so to bin/Android/arm64
 ** BUILD SUCCEEDED (Android) **
 ```
 
-At this stage, you should have 3 libraries in the **bin** folder under **Android/{target_architecture}**. These should be visible within the **Explorer** view alongside our source code, build, and environment setting files. We will address this later.
+At this stage, you should have 4 libraries in the **bin** folder under **Android/{target_architecture}**. These should be visible within the **Explorer** view alongside our source code, build, and environment setting files. We will address this later.
```

Comment:
This line confused me when I was going through the tutorial, since I had binaries for `arm`, `arm64`, `x86`, and `x86_64`.  A later line in the tutorial made it clear that this was correct, but this caught me off guard for a moment.

Otherwise, I didn't notice any typos or mistakes.  Excellent work!  This tutorial is an invaluable resource!